### DISCUSSION TITLE
force-pyramid: "parallel": external command not found

### DISCUSSION THREAD

**Discussion by Antguz** [davidfrantz/force#327] [Q&A] [answered]
Dear community,

I have found an unexpected behavior in the module of `force-pyramid`. When I run it on level3 data, I got a message that says `force-pyramid: "parallel": external command not found`. I got the same message even using the help option `force-pyramid -h`. I can not even reproduce the tutorials that use the pyramid module. 

Am I doing something wrong? 

I am running the last version through singularity.

AG.

**Comment by davidfrantz** [comment, marked as answer]
Dear AG,

we have made some changes in the underlying base image, which might have caused temporary unavailability of said program.

I suspect that you pulled / converted the image at an unfavorable point in time. Could you please try again, i.e. building a new image?

Best,
David

**Reply by Antguz** [reply]
It works now! Thanks
